content/en/docs/releases/supported-releases/index.md

```diff
@@ -51,16 +51,17 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported  | Release Date      | End of Life            | Supported Kubernetes Versions | Tested, but not supported |
-|-----------------|----------------------|-------------------|------------------------|-------------------------------|---------------------------|
-| master          | No, development only |                   |                        |                               |                           |
-| 1.12            | Yes                  | November 18, 2021 | ~June 2022 (Expected)  | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
-| 1.11            | Yes                  | August 12, 2021   | ~Mar 2022 (Expected)   | 1.18, 1.19, 1.20, 1.21, 1.22  | 1.16, 1.17                |
-| 1.10            | No                   | May 18, 2021      | Dec 30, 2021           | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17, 1.22          |
-| 1.9             | No                   | February 9, 2021  | Oct 8, 2021            | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                |
-| 1.8             | No                   | November 10, 2020 | May 12, 2021           | 1.16, 1.17, 1.18, 1.19        | 1.15                      |
-| 1.7             | No                   | August 21, 2020   | Feb 25, 2021           | 1.16, 1.17, 1.18              | 1.15                      |
-| 1.6 and earlier | No                   |                   |                        |                               |                           |
+| Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported |
+|-----------------|----------------------|-------------------|--------------------------|-------------------------------|---------------------------|
+| master          | No, development only |                   |                          |                               |                           |
+| 1.13            | Yes                  | February 8, 2022  | ~October 2022 (Expected) | 1.19, 1.20, 1.21, 1.22, 1.23  | 1.16, 1.17, 1.18          |
```

Comment:
Does 1.19 fall off here?

Reply:
I suspect 1.19 would be removed, but this list usually, IIRC, shows up on the TOC agenda. Since k8s 1.24 doesn't come out until the end of April, I suspect it won't show up as supported in Istio 1.13, but would for 1.14, but that's just a thought on my part (as we haven't listed 1.23 for 1.12 and those release dates were closer).

Reply:
Lets move 1.19 to "tested but not supported" IMO

Reply:
+1

Reply:
@jacob-delgado is missing a `, ` in his suggestion.